src/coreclr/nativeaot/Runtime/RhConfig.cpp

```diff
@@ -179,6 +191,103 @@ void RhConfig::ReadEmbeddedSettings()
     return;
 }
 
+bool RhConfig::GetEmbeddedKnob(_In_z_ const char* configName, _Out_ const char** configValue)
```

Comment:
The new methods look very similar to the existing ones. Can we share them?

Reply:
If we share, we'd delete the concept of `RCV_Count`.

Reply:
Note that there's another PR in flight making changes here so we might want to pause for a moment to avoid conflicts. #86656 cc @elinor-fung

Reply:
I do not sure if sharing make things better, but please take a look.